tools/prepare_release.py

```diff
@@ -10,12 +10,15 @@
 import subprocess
 from collections import defaultdict
 from dataclasses import dataclass
-from typing import Generator
+from typing import TYPE_CHECKING
 
 import click
 import httpx
 import msgspec
 
+if TYPE_CHECKING:
```

Comment:
Sorry, why is this import under `TYPE_CHECKING`?

Reply:
No idea. That's something ruff did automatically :shrug:

Reply:
this does not have any runtime benefits, because we already import `typing` and `typing` imports `collections.abc`, so I guess it is safe to simplify the code. (moreover, we don't really care about import-time speed in a `tools/` file)

Reply:
Yeah, I know it doesn't really make sense. But I don't want to fight with ruff. I prefer just letting it do what it feels like doing, so I don't have to think about it :)

Reply:
sounds good to me!